homeassistant/components/influxdb.py

```diff
@@ -92,50 +89,56 @@ def setup(hass, config):
     def influx_event_listener(event):
         """Listen for new messages on the bus and sends them to Influx."""
         state = event.data.get('new_state')
-        if state is None or state.state in (
-                STATE_UNKNOWN, '', STATE_UNAVAILABLE) or \
-                state.entity_id in blacklist:
+        if state is None or state.entity_id in blacklist:
             return
 
-        try:
-            if len(whitelist) > 0 and state.entity_id not in whitelist:
-                return
+        if len(whitelist) > 0 and state.entity_id not in whitelist:
+            return
 
+        try:
             _state = state_helper.state_as_number(state)
         except ValueError:
             _state = state.state
 
-        measurement = state.attributes.get('unit_of_measurement')
-        if measurement in (None, ''):
-            if default_measurement:
-                measurement = default_measurement
-            else:
-                measurement = state.entity_id
-
+        # Create a counter for this state change
         json_body = [
             {
-                'measurement': measurement,
+                'measurement': "hass.state.count",
                 'tags': {
                     'domain': state.domain,
                     'entity_id': state.object_id,
                 },
                 'time': event.time_fired,
                 'fields': {
-                    'value': _state,
+                    'value': 1
                 }
             }
         ]
 
+        json_body[0]['tags'].update(tags)
+
+        state_fields = {}
+        if isinstance(_state, (int, float)):
```

Comment:
I'm actually okay with this allowing booleans of others are okay with it. If needed I can block them though.